src/providers/krb5/krb5_child.c

```diff
@@ -1087,7 +1088,7 @@ static errno_t pack_response_packet(TALLOC_CTX *mem_ctx, errno_t error,
 
     buf = talloc_array(mem_ctx, uint8_t, size);
     if (!buf) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Insufficient memory to create message.\n");
+        DEBUG(SSSDBG_CRIT_FAILURE, "talloc_array() failed\n");
```

Comment:
In other places in this file when function name is mentioned in log it is printed without brackets:
"talloc_array failed\n"

Reply:
Ok, fixed.